manifests/init.pp

```diff
@@ -52,6 +54,7 @@
 #
 class foreman_proxy_content (
   String[1] $parent_fqdn = $foreman_proxy_content::params::parent_fqdn,
+  String[1] $foreman_fqdn = $foreman_proxy_content::params::foreman_fqdn,
```

Comment:
```foreman_proxy``` has a parameter ```foreman_base_url``` and I'm wondering if it should be passed to it, but that would mean changing the ```include``` to a ```class``` and that might break some things.